client/controllers/incidentModal.coffee

```diff
@@ -22,8 +22,6 @@ Template.incidentModal.events
            $(form.count).val("")
            $(form.incidentType).val(null).trigger("change")
            $(form.travelRelated).attr('checked', false)
-        if closeModal
```

Comment:
I was getting an error because closeModal wasn't defined. Should the modal be closed after "save" is clicked?